### PR TITLE
Changes with entrypoint to get working with GCP 

### PR DESCRIPTION
**Pull request by jnsproul**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
-version: '3.8'
+version: "3.8"
```
Comment (cbcunc) [inline docker-compose.yml]: Version is a deprecated docker compose keyword now. It can and should be removed.

**Review by cbcunc — CHANGES_REQUESTED** [review]
Take a look at the one liner change requested for docker-compose.yml. I am also going to refrain from approving this PR until after the meeting tomorrow anyway because I want Mauriel to see it also.

**Review by Maurier — APPROVED** [review]
Looks good. I had a look at this PR and I think these environment variable mappings are correct.

**Comment by cbcunc** [comment]
Meeting moved to bi-weekly. So this will be approved with the one-liner change is in place. Thanks.

File: docker-compose.yml (hunk 1 above)
Comment (jnsproul) [reply docker-compose.yml]: Completed with [7656bdf](https://github.com/cznethub/cznet-discovery/pull/36/commits/7656bdfd0577d70387ae981abdfd4e912cddb98a)

**Review by cbcunc — APPROVED** [review]

**Pull request merged**